zerver/openapi/zulip.yaml

```diff
@@ -9086,25 +9116,15 @@ paths:
                         "msg": "",
                         "presences":
                           {
-                            "iago@zulip.com":
+                            "10":
```

Comment:
Yes, that sounds correct. Given that `slim_presence` has been available since 2020, I think it's safe to not put too much effort into documenting the legacy format, which will die when we switch away from the old mobile apps.